docs/developer/Releases.md

```diff
@@ -0,0 +1,117 @@
+Release Approach
```

Comment:
I would use the naming convention that we used with the other documentation markdownsand call this `releases.md`, with the first letter in lowercase